st2common/st2common/content/loader.py

```diff
@@ -268,3 +269,124 @@ def _load(self, parser_func, file_path):
             except ParserError:
                 LOG.exception("Failed loading content from %s.", file_path)
                 raise
+
+
+class OverrideLoader(object):
+    """
+    Class for loading pack override data
+    """
+
+    # Mapping of permitted override types to resource name
+    ALLOWED_OVERRIDE_TYPES = {
+        "sensors": "class_name",
+        "actions": "name",
+        "rules": "name",
+        "aliases": "name",
+    }
+
+    ALLOWED_OVERRIDE_NAMES = [
+        "enabled",
+    ]
+
+    def override(self, pack_name, type, content):
```

Comment:
Would it be better to avoid hijacking the `type` keyword and use `resource_type` instead?